src/components/layout.jsx

```diff
@@ -5,7 +5,7 @@ export default function Layout({ children }) {
   const style = {
     margin: `2rem auto`,
     maxWidth: 650,
-    padding: `2rem 2rem`,
+    padding: `5px 5px`,
```

Comment:
Change this to `0.5rem`
```suggestion
    padding: `0.5rem`,
```